library/query_self_hosted_workflow.yml

```diff
@@ -0,0 +1,25 @@
+id: RP-10
+
+info:
+  name: Self Hosted Run
+  severity: high
```

Comment:
I would reduce this to medium.
This is the legitimate behavior for many opensource projects